plugins/org.wso2.developerstudio.eclipse.distribution.project/src/org/wso2/developerstudio/eclipse/distribution/project/ui/wizard/DistributionProjectWizard.java

```diff
@@ -113,20 +129,28 @@ public boolean performFinish() {
 			Xpp3Dom serverOperation = MavenUtils.createXpp3Node(carbonServer, "operation");
 			serverOperation.setValue("deploy");
 
+			log.info("1");

```

Comment:
1. What is the meaning of this INFO log? Who knows what is "1,2, 3, ...."? If you are adding logs, add meaningful logs or don't add these sort of logs. 
2. NOT INFO logs. Change to DEBUG.